gubernator/filters_test.py

```diff
@@ -45,6 +45,17 @@ def test_linkify(self):
     def test_slugify(self):
         self.assertEqual('k8s-test-foo', filters.do_slugify('[k8s] Test Foo'))
 
+    def test_testcmd_unit(self):
+        pkg = 'k8s.io/kubernetes/pkg/api/errors'
+        name = 'TestErrorNew'
+        expected = 'go test -v %s -run %s$' % (pkg, name)
```

Comment:
generic testing blech: I'd rather you just copy the strings a few times than duplicate logic between the test and the code under test.